verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -110,23 +83,34 @@ void KytheFactsExtractor::IndexingFactNodeTagResolver(
     }
     case IndexingFactType::kFunctionOrTask: {
       vname = ExtractFunctionOrTask(node);
-      scope_context_.top().push_back(vname);
+      break;
+    }
+    case IndexingFactType::kDataTypeReference: {
```

Comment:
Nice change, with more consistent case style handling of CST nodes.

Reply:
Thanks :)